### DISCUSSION TITLE
Can Workflows add a activity that can popup form to allow users to fill in data,please?

### DISCUSSION THREAD

**Discussion by effapp** [OrchardCMS/OrchardCore#14900] [General]
Can Workflows add a activity that can popup form to allow users to fill in data,please?

It would be best to give an example code，Thanks.

**Comment by sebastienros** [comment]
You can redirect users to a form. Does the "popup" effect matter?

**Comment by effapp** [comment]
> You can redirect users to a form. Does the "popup" effect matter?

Hi:
      Because "popup" scences frequently appear.

**Comment by sebastienros** [comment]
No it can't
